bot/exts/evergreen/emojis.py

```diff
@@ -0,0 +1,39 @@
+import textwrap
+
+from discord import Color, Embed, Emoji
+from discord.ext import commands
+
+from bot.utils.time import time_since
+
+
+class Emojis(commands.Cog):
+    """Has commands related to emojis."""
```

Comment:
I'd love to see this explained a bit further. What does it actually do? All info in the docstring is already given by the class name.

Reply:
I don't know, suggest something lol

Reply:
Think about what the commands actually do. Feel free to take inspiration from the other docstrings in the repo.